src/domain/event/eventNavigation/EventNavigation.tsx

```diff
@@ -0,0 +1,115 @@
+import classNames from 'classnames';
+import { css } from 'emotion';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Button from '../../../common/components/button/Button';
+import Container from '../../app/layout/Container';
+import { useTheme } from '../../app/theme/Theme';
+import styles from './eventNavigation.module.scss';
+
+interface EventNavigationItem {
+  component: React.ReactElement;
+  disabled?: boolean;
+  isCompleted: boolean;
+  label: string;
+}
+
+interface Props {
+  items: EventNavigationItem[];
+}
+
+const EventNavigation: React.FC<Props> = ({ items }) => {
+  const { t } = useTranslation();
+  const { theme } = useTheme();
+  const [activeTab, setActiveTab] = React.useState(0);
+
+  const handleItemClick = (item: EventNavigationItem, index: number) => () => {
+    if (item.disabled) return;
+
+    setActiveTab(index);
+  };
+
+  const isPreviousDisabled = !activeTab;
+  const isNextDisabled: boolean =
+    activeTab === items.length - 1 || !!items[activeTab + 1]?.disabled;
+
+  const previous = () => {
```

Comment:
handlePreviousClick might be more fitting name for this

Reply:
Fixed